src/mudlet.h

```diff
@@ -722,6 +730,10 @@ private slots:
 
     // Whether multi-view is in effect:
     bool mMultiView;
+
+    // read-only value to see if the interface is light or dark. To set the value,
```

Comment:
"read-only" is not the correct term IMHO, perhaps you meant to emphasize that it was `private` and should not be set directly, but instead via the *setter* `setAppearance(...)` ?